src/mgmt/cli_query.c

```diff
@@ -124,3 +124,213 @@ dc_mgmt_get_bs_state(tse_task_t *task)
 	tse_task_complete(task, rc);
 	return rc;
 }
+
+struct mgmt_reset_led_state_arg {
+	struct dc_mgmt_sys	*sys;
+	crt_rpc_t		*rpc;
+};
+
+static int
+mgmt_reset_led_state_cp(tse_task_t *task, void *data)
+{
+	struct mgmt_reset_led_state_arg	*arg;
+	struct mgmt_reset_led_state_out	*rl_out;
+	struct mgmt_reset_led_state_in	*rl_in;
+	int				 rc = task->dt_result;
+
+	arg = (struct mgmt_reset_led_state_arg *)data;
+
+	if (rc) {
+		D_ERROR("RPC error while resetting LED state: "DF_RC"\n",
+			DP_RC(rc));
+		D_GOTO(out, rc);
+	}
+
+	rl_out = crt_reply_get(arg->rpc);
+	D_ASSERT(rl_out != NULL);
+	rc = rl_out->rl_rc;
+	if (rc) {
+		D_ERROR("MGMT_RESET_LED_STATE replied failed, rc: %d\n", rc);
+		D_GOTO(out, rc);
+	}
+
+
+	rl_in = crt_req_get(arg->rpc);
+	D_ASSERT(rl_in != NULL);
+
+out:
+	dc_mgmt_sys_detach(arg->sys);
+	crt_req_decref(arg->rpc);
+
+	return rc;
+}
+
+int
+dc_mgmt_reset_led_state(tse_task_t *task)
+{
+	daos_mgmt_reset_led_state_t	*args;
+	crt_endpoint_t			 svr_ep;
+	crt_rpc_t			*rpc_req = NULL;
+	crt_opcode_t			 opc;
+	struct mgmt_reset_led_state_in	*rl_in;
+	struct mgmt_reset_led_state_arg	 cb_args;
+	int				 rc;
+
+	args = dc_task_get_args(task);
+
+	rc = dc_mgmt_sys_attach(args->grp, &cb_args.sys);
+	if (rc != 0) {
+		D_ERROR("cannot attach to DAOS system: %s\n", args->grp);
+		D_GOTO(out, rc);
+	}
+
+	svr_ep.ep_grp = cb_args.sys->sy_group;
+	svr_ep.ep_rank = 0;
```

Comment:
I don't quite follow this, this request will always be sent to rank 0? How will rank 0 forward the request to the proper rank having the requested device?